schema/admin-stats-response.json

```diff
@@ -115,6 +115,14 @@
                 "checksum": {
                     "type": "number"
                 },
+                "checksums": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^.*$": {
+                            "type": "number"
```

Comment:
The result of a `farmhash` is always a number. That's the bit I was missing (I thought it was alphanumeric). Cool, thanks!